tests/test_custom_checkers/test_top_level_code_checker.py

```diff
@@ -86,6 +86,15 @@ def test_no_message_constant_assignment(self):
         with self.assertNoMessages():
             self.checker.visit_module(mod)
 
+    def test_no_message_annotated_constant_assignment(self):
```

Comment:
This is a good test to add. Since this is a "no message" case, I suggest adding a second test which is a "reports message" case when the variable isn't in `ALL_CAPS` naming style.